oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Configuration.java

```diff
@@ -320,6 +321,13 @@
                     " (milliseconds).")
     long recoveryDelayMillis() default DEFAULT_RECOVERY_DELAY_MILLIS;
 
+    @AttributeDefinition(
+            name = "Perflogger Info Millis",
+            description = "Minimum duration (in milliseconds) for an operation that perflogger info will log. " +
```

Comment:
```suggestion
            description = "Minimum duration (in milliseconds) for certain operations that perflogger info will log. " +
```
perhaps we can emphasize that it is only "certain" operations (while not making a definition of what certain exactly means, that can also change over time) are perf-logged. otherwise it might imply that perflogger in general would use this (which might or might not be the case)

Reply:
Done.